pkgs/by-name/ht/httptoolkit-server/package.nix

```
@@ -0,0 +1,146 @@
+{
+  lib,
+  stdenv,
+  nodejs_20,
+  buildNpmPackage,
+  fetchFromGitHub,
+  fetchzip,
+  writeShellScriptBin,
+  nss,
+}:
+
+let
+  nodejs = nodejs_20;
+  buildNpmPackage' = buildNpmPackage.override { inherit nodejs; };
```

```suggestion
  buildNpmPackage' = buildNpmPackage.override { nodejs = nodejs_20; };
```
Since you're not using `nodejs` anywhere else this should be fine.